NuRadioReco/utilities/noise.py

```diff
@@ -548,13 +497,6 @@ def generate_noise2(self, debug=False):
 
                 traces[iCh] = perfect_floor_comparator(trace, self.adc_n_bits, self.adc_ref_voltage)
 
-            phased_traces = np.zeros((len(self.beam_time_delays), self.n_samples * self.upsampling))
```

Comment:
was this really unused?

Reply:
I think you had started implementing the (anyway much slower) generate_noise2 from the `generate_noise` and this was a leftover. In this function only  the 1D `phased_trace` is used iteratively in the nested loops to sum up the `shifted_traces`